boards/nucleo-f401/include/board.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2015 Lari Lehtomäki
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    boards_nucleo-f401 Nucleo-F401
+ * @ingroup     boards
+ * @brief       Board specific files for the nucleo-f401 board
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the nucleo-f401 board
+ *
+ * @author      Lari Lehtomäki <lari@lehtomaki.fi>
+ */
+
+#ifndef BOARD_H_
+#define BOARD_H_
+
+#include "cpu.h"
+#include "periph_conf.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name Define the nominal CPU core clock in this board
+ */
+#define F_CPU               CLOCK_CORECLOCK
+
+/**
+ * @name xtimer configuration
+ * @{
+ */
+#define XTIMER              TIMER_0
+#define XTIMER_CHAN         (0)
+#define XTIMER_OVERHEAD     (6)
+#define XTIMER_BACKOFF      (5)
+/** @} */
+
+/**
+ * @name Define UART device and baudrate for stdio
+ * @{
+ */
+#define STDIO               UART_0
+#define STDIO_BAUDRATE      (115200U)
+#define STDIO_RX_BUFSIZE    (64U)
+/** @} */
+
+/**
+ * @name LED pin definitions
+ * @{
+ */
+#define LED_GREEN_PORT      (GPIOA)
+#define LED_GREEN_PIN       (5)
+#define LED_GREEN_GPIO      GPIO_PIN(PORT_A,5)
```

Comment:
I'm not really following what you mean with this.
PORT_A is defined in [cpu/stm32f4/include/periph_cpu.h#L50](https://github.com/RIOT-OS/RIOT/blob/master/cpu/stm32f4/include/periph_cpu.h#L50)

And [DM00105823](http://www.st.com/web/en/resource/technical/document/user_manual/DM00105823.pdf#21) mentions on page 17 on section 5.4 LEDs that green is on pin PA5.

Or did I understand you completely wrong? :smiley: